package-lock.json

```diff
@@ -8617,55 +10139,519 @@
       }
     },
     "webpack-dev-server": {
-      "version": "1.14.0",
```

Comment:
Hmmm, this is a little odd. Your requirement is `^3.1.5`, but it looks like the version installed is `1.14.0`. That's why Dependabot thought this was an update.

Reply:
Oh I see...

Reply:
Please take a look at #206

Reply:
Looks like that fixes things!